components/tdisplays3/t_display_s3.cpp

```diff
@@ -46,15 +152,63 @@ int TDisplayS3::get_height_internal() {
   }
 }
 
+void TDisplayS3::updateArea(int16_t x1, int16_t y1, int16_t x2, int16_t y2, uint16_t *buffer, void (*ready_callback)(void)) {
+  if (!powered_on_) return;
+
+#ifdef TDISPLAYS3_USE_ASYNC_IO
+    if (writer_shim::writeFrameTask_ == nullptr) {
+      return;
+    }    
+    writer_shim::startWrite(x1, y1, x2, y2, buffer, ready_callback);
+#else
+    this->tft_->pushImage(x1, y1, (x2 - x1) + 1, (y2 - y1) + 1, buffer);
+   ready_callback();
+#endif
+
+}
+
 void TDisplayS3::update() {
+  if (!powered_on_) return;
+
+  //Let lambdas draw their last stuff
   this->do_update_();
-  this->spr_->pushSprite(0, 0);
+
+#ifdef TDISPLAYS3_USE_ASYNC_IO
+
+    if (this->disable_buffer_) {
+      return;
+    }
+
+    //Grab and swap screen-buffers
+    auto currentBuffer = this->buffer_;
+    if (((uint8_t*) this->spr_->frameBuffer(1)) != currentBuffer) {
+      this->buffer_ = (uint8_t*) this->spr_->frameBuffer(1);
+    } else {
+      this->buffer_ = (uint8_t*) this->spr_->frameBuffer(2);
+    }
+
+    this->updateArea((int16_t) 0, (int16_t) 0, this->spr_->width(), this->spr_->height(), (uint16_t*) this->buffer_, nullptr);
+#else
+    this->spr_->pushSprite(0, 0);  
+#endif
+
 }
 
 void TDisplayS3::set_dimensions(uint16_t width, uint16_t height) {
   this->width_ = width;
   this->height_ = height;
 }
 
+void TDisplayS3::power_off() {
+  this->tft_->writecommand(ST7789_DISPOFF); //turn off lcd display
+  this->powered_on_ = false;
+}
```

Comment:
No, these are 2 different things. In my battery powered application I have the background-led wired to a light-output, so I can set the brightness trough PWM. Sometimes, I fade to back when the user is not doing anything, but will restore to normal backlight when the user touches the display. In the meantime, the display is staying up-to-date with the data, but you won't be able to see it due to the backlight being off.

Powering down the display chip is a whole different beast. It will stop accepting commands and loose it's memory, and will go to a very low power state entirely. A reset is needed to bring it back to life again.

All this stuff is crucial when you don't want to drain your lipo battery during power-off.